eras/allegra/impl/cardano-ledger-allegra.cabal

```
@@ -53,7 +53,7 @@ library
         cardano-crypto-class,
         cardano-ledger-binary >=1.0,
         cardano-ledger-core ^>=1.3,
-        cardano-ledger-shelley ^>=1.3,
+        cardano-ledger-shelley ^>=1.4,
```

```suggestion
        cardano-ledger-shelley >=1.3 && <1.5,
```
Also needs a patch version bump